hack/test-cmd.sh

```diff
@@ -192,8 +192,8 @@ fi
 # must only accept one arg (server)
 [ "$(oc login https://server1 https://server2.com 2>&1 | grep 'Only the server URL may be specified')" ]
 # logs in with a valid certificate authority
-oc login ${KUBERNETES_MASTER} --certificate-authority="${MASTER_CONFIG_DIR}/ca.crt" -u test-user -p anything --api-version=v1beta3
-grep -q "v1beta3" ${HOME}/.kube/config
+oc login ${KUBERNETES_MASTER} --certificate-authority="${MASTER_CONFIG_DIR}/ca.crt" -u test-user -p anything --api-version=v1
+grep -q "v1" ${HOME}/.kube/config
```

Comment:
we need to make sure the login command still supports the --api-version arg, and stores the correct api version for the server